document_url/README.rst

```diff
@@ -40,7 +40,9 @@ Contributors
 ------------
 
 * Jonathan Nemry <jonathan.nemry@acsone.eu>
-* Pedro M. Baeza <pedro.baeza@tecnativa.com
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Nicolás Ramos <contacto@difusionvisual.com>
+* Antonio Cánovas <antonio.canovas@ingenieriacloud.com>
```

Comment:
I have created a PR to improve the template to add the possibility of defining funders: https://github.com/OCA/maintainer-tools/pull/241